### ISSUE TITLE
All text is being shifted down - especially svg icons

### ISSUE THREAD

**Issue by samvk** [h-ibaldo/Raleway_Fixed_Numerals#7]
When switching from Raleway to Rawline, all of the text is shifted down a several pixels.
This also seems to have an extra effect on svg icon sets, which is moving down significantly.

![1-raleway](https://user-images.githubusercontent.com/12996081/30971045-e14ba296-a434-11e7-8d4a-6c0c61c7a71d.png)
![2-rawline](https://user-images.githubusercontent.com/12996081/30971051-e5f25c54-a434-11e7-9399-f6d60870a6b5.png)

If you switch back and force between these images of each font you can see all the text shifts down, as well as two top-right svgs move down significantly.

**Comment by caitlinwynne** [comment]
Yes, we are having the same problem. See the attached image, the icon and the text are not aligning anymore. This is through-out the whole site. We would really like to continue using Rawline but need to fix the line height issue. Please help! Thank you.
![screen shot 2017-10-05 at 11 30 43 am](https://user-images.githubusercontent.com/32529960/31206015-1d132574-a9c1-11e7-9efb-ded063fb2fd5.png)

**Comment by caitlinwynne** [comment]
@samvk Did you figure out a solution?

**Comment by MrChriZ** [comment]
I thought i was going nuts!
Here's a codepen demonstrating the issue:
https://jsfiddle.net/beudn8fn/

**Comment by MrChriZ** [comment]
@bramp has a better solution here:
https://blog.bramp.net/post/2018/01/21/google-font-features/

Here's a fiddle which shows the alignment is correct:
https://jsfiddle.net/e9rnwkbv/

**Comment by connorsmallman** [comment]
@h-ibaldo any ideas on how to go about fixing this issue? Happy to open a pr if you could point me in the right direction

**Comment by maxim25** [comment]
Any update on this? I have the same issue, event tried to keep lnum with font-squirrel myself but ran into the same issue.

**Comment by samvk** [comment]
If it helps anyone, I just ended up making my own version: [Monorale: Raleway Sober](https://github.com/samvk/monorale-raleway-sober).

Only problem is I ended up losing ligatures in the process (I also overwrote the "fancy" `w` with the normal one out of preference.) But if anyone wants to try the same thing or make a fork, just try editing the original Raleway and overwriting the "oldstyle" numerals with the normal ones (they already come with Raleway as alternates.)
